pyproject.toml

```diff
@@ -51,7 +51,7 @@ documentation = "https://docs.saleor.io/"
   jaeger-client = "^4.5.0"
   openpyxl = "^3.0.3"
   django-cache-url = "^3.1.2"
-  pyjwt = ">=2.4.0,<3.0.0"
+  pyjwt = "2.5.0"
```

Comment:
Add a comment why it's locked + GitHub issue link

Reply:
It is commented, I will create a issue for that